crates/router/src/routes/app.rs

```diff
@@ -203,26 +203,20 @@ impl Payouts {
 
 pub struct PaymentMethods;
 
-#[cfg(any(feature = "olap", feature = "oltp"))]
+#[cfg(feature = "oltp")]
 impl PaymentMethods {
     pub fn server(state: AppState) -> Scope {
-        let mut route = web::scope("/payment_methods").app_data(web::Data::new(state));
-        #[cfg(feature = "oltp")]
-        {
-            route = route
-                .service(web::resource("").route(web::post().to(create_payment_method_api)))
-                .service(
-                    web::resource("/{payment_method_id}")
-                        .route(web::get().to(payment_method_retrieve_api))
-                        .route(web::post().to(payment_method_update_api))
-                        .route(web::delete().to(payment_method_delete_api)),
-                );
-        }
-        #[cfg(feature = "olap")]
-        {
-            route = route.service(web::resource("").route(web::get().to(list_payment_method_api)));
-        }
-        route
+        web::scope("/payment_methods")
+            .app_data(web::Data::new(state))
+            .service(web::resource("").route(web::post().to(create_payment_method_api)))
+            .service(
+                web::resource("/{payment_method_id}")
+                    .route(web::get().to(payment_method_retrieve_api))
+                    .route(web::post().to(payment_method_update_api))
+                    .route(web::delete().to(payment_method_delete_api)),
+            )
+            .service(web::resource("").route(web::get().to(list_payment_method_api)))
```

Comment:
Merge lines 211 and 218, else it could have unintended effects because of the order in which paths are registered.

Reply:
One is post other is get request, It wouldn't right ?